jkube-kit/enricher/specific/src/test/java/org/eclipse/jkube/kit/enricher/specific/AbstractHealthCheckEnricherTest.java

```diff
@@ -41,9 +42,13 @@
  */
 public class AbstractHealthCheckEnricherTest {
 
-    @Mocked
     KitLogger log;
 
+    @Before
+    public void setUp() throws Exception {
+        log = mock(KitLogger.class);
```

Comment:
Let's try to use the same pattern we've described in other PRs. If there are verifications, a combination of spy+SilentLogger. I there aren't, a plain SilentLogger instance.